scripts/projects.js

```diff
@@ -1,22 +1,44 @@
 'use strict';
-var projects = [];
+// var projects = [];
 
 function Project(projData){
   this.name = projData.name,
   this.ref = projData.ref;
 };
 
+Project.all = [];
+
 Project.prototype.toHtml = function () {
   var source = $('#project-template').text();
   var templateRender = Handlebars.compile(source);
-
+  console.log(source);
```

Comment:
**TODO FOR LOST POINTS**

Remember to take out console.logs before submitting. Console.logs are great for debugging as I did with you, but just remember to get rid of them when we get something working properly.